psi4/driver/procrouting/response/scf_response.py

```diff
@@ -782,7 +782,7 @@ def tda_solver(e, n, g, m):
                                x.L_eigvec if restricted else x.L_eigvec[0])
```

Comment:
Could you please fix the indentation of the first `set_array_variable` such that it is consistent with the other three? 👍🏻

Reply:
Sure, done in a08d574ed07ddd31b266b80293df739703e03f98